### PR TITLE
Tweak docs: constants, blocks and so on

### PR DESCRIPTION
**Pull request by brusherru**
There is no issue.

### CODE REVIEW

File: docs/reference/node-cpp-api/README.md [hunk 1]
```diff
@@ -29,31 +39,73 @@ Required as an argument for most node functions. Can be thought as an explicit `
...
+If a node has multiple unlabeled outputs, they are referred as `output_OUT1`, `output_OUT2`,…, `output_OUT7`.
 
-<a name="typeof_xxx"></a>
+### Constant pin types
 
-#### `typeof_$$$`
+#### Port
 
-A type for each of node's pins. Useful to access the type of a pin if its symbol is hardly accessible in other ways.
+Port pin type is a constant type of `uint8_t`.
```
Comment (evgenykochetkov) [inline docs/reference/node-cpp-api/README.md:76]: I'd move this somewhere in `/docs/guide/data-types`

Comment (brusherru) [reply docs/reference/node-cpp-api/README.md:76]: 👌

File: docs/reference/node-cpp-api/README.md [hunk 2]
```diff
@@ -19,6 +19,16 @@ Represents a big integer to store time values expressed in milliseconds. Typedef
 
 Represents XOD string. Typedef’ed to [`List<char>`](#List), so the same methods and functions may be used to manipulate it.
 
+## Inside `node {}`
+
+The `node` block represents contents of a `struct Node`.
```
Comment (evgenykochetkov) [inline docs/reference/node-cpp-api/README.md:24]:
```suggestion
The `node` block describes the state and behaviour of a node. Under the hood, it is converted to a regular C++ `struct`, and should be treated as such. You can add fields and methods to it.
```

File: docs/reference/node-cpp-api/README.md [hunk 3]
```diff
@@ -29,31 +39,66 @@ Required as an argument for most node functions. Can be thought as an explicit `
...
+`$$$` should be replaced with a valid value for the output.
+
+In case of creating an "unpack" node for the custom type, which is a struct that contains a constant value (such as PORT), you might define it as:
+```cpp
+static constexpr typeof_OUT constant_output_OUT = typeof_DEV::port;
+```
+
+## `nodespace {}`
+
+The `nodespace` block used to place some code outside the `struct Node {}`, but inside a node namespace, so this code will be accessible only with that node.
```
Comment (evgenykochetkov) [inline docs/reference/node-cpp-api/README.md:99]:
```suggestion
The `nodespace` block is used to place some code outside the `struct Node {}`, but inside a node namespace, so this code will be accessible only with that node.
```

**Review by evgenykochetkov — APPROVED** [review]

**Pull request merged**